tests/Functional/Command/ModifyEntity/LineItem/LineItemChangeDatesCommandTest.php

```diff
@@ -215,7 +214,8 @@ public function provideInvalidParameters(): array
                     '--start-date' => '2020-01-01T00:00:00+0000',
                     '--end-date' => '2020-01-10T00:00:00+0000',
                 ],
-                'expectedOutput' => 'You need to specify line-item-ids or line-item-slugs option.',
+                'expectedOutput' => 'You need to specify \'line-item-ids\', \'line-item-slugs\'' .
+                    ' or \'line-item-group-ids\' option.',
```

Comment:
Can you please use double quotes in the outside, so you can spare the escapings

Reply:
Fixed! Thanks!